static/usage/v7/theming/automatic-dark-mode/javascript.md

```diff
@@ -1,105 +1,86 @@
-            </ion-button>
-          </ion-buttons>
-        </ion-toolbar>
-      </ion-header>
+  <ion-list-header>Brightness</ion-list-header>
+  <ion-list inset="true">
+    <ion-item>
+      <ion-range value="40">
+        <ion-icon name="sunny-outline" slot="start"></ion-icon>
+        <ion-icon name="sunny" slot="end"></ion-icon>
+      </ion-range>
+    </ion-item>
+    <ion-item>
+      <ion-toggle justify="space-between" checked>True Tone</ion-toggle>
+    </ion-item>
+  </ion-list>
 
-      <ion-content>
-        <ion-list-header>Appearance</ion-list-header>
-        <ion-list inset="true">
-          <ion-item button="true">Text Size</ion-item>
-          <ion-item>
-            <ion-toggle justify="space-between">Bold Text</ion-toggle>
-          </ion-item>
-        </ion-list>
+  <ion-list inset="true">
+    <ion-item button="true">
+      <ion-label>Night Shift</ion-label>
+      <ion-text slot="end" color="medium">9:00 PM to 8:00 AM</ion-text>
+    </ion-item>
+  </ion-list>
+</ion-content>
 
-        <ion-list-header>Brightness</ion-list-header>
-        <ion-list inset="true">
-          <ion-item>
-            <ion-range value="40">
-              <ion-icon name="sunny-outline" slot="start"></ion-icon>
-              <ion-icon name="sunny" slot="end"></ion-icon>
-            </ion-range>
-          </ion-item>
-          <ion-item>
-            <ion-toggle justify="space-between" checked>True Tone</ion-toggle>
-          </ion-item>
-        </ion-list>
+<script>
+  // Use matchMedia to check the user preference
+  const prefersDark = window.matchMedia('(prefers-color-scheme: dark)');
 
-        <ion-list inset="true">
-          <ion-item button="true">
-            <ion-label>Night Shift</ion-label>
-            <ion-text slot="end" color="medium">9:00 PM to 8:00 AM</ion-text>
-          </ion-item>
-        </ion-list>
-      </ion-content>
+  toggleDarkTheme(prefersDark.matches);
 
-      <script>
-        // Use matchMedia to check the user preference
-        const prefersDark = window.matchMedia('(prefers-color-scheme: dark)');
+  // Listen for changes to the prefers-color-scheme media query
+  prefersDark.addEventListener('change', (mediaQuery) => toggleDarkTheme(mediaQuery.matches));
 
-        toggleDarkTheme(prefersDark.matches);
+  // Add or remove the "dark" class on the document body
+  function toggleDarkTheme(shouldAdd) {
+    document.body.classList.toggle('dark', shouldAdd);
+  }
+</script>
 
-        // Listen for changes to the prefers-color-scheme media query
-        prefersDark.addEventListener('change', (mediaQuery) => toggleDarkTheme(mediaQuery.matches));
-
-        // Add or remove the "dark" class on the document body
-        function toggleDarkTheme(shouldAdd) {
-          document.body.classList.toggle('dark', shouldAdd);
-        }
-      </script>
-
-      <style>
-        /*
+<style>
+  /*
       * Optional CSS
       * -----------------------------------
       */
```

Comment:
Can this be outdented? It doesn't align right in the code block:

<img width="678" alt="Screenshot 2023-09-18 at 11 11 40 AM" src="https://github.com/ionic-team/ionic-docs/assets/6577830/a50ad691-1eff-4823-a412-acb1d50d0a98">